package.json

```diff
@@ -12,6 +12,8 @@
     "lingui:extract-and-check": "node localebuilder.js --check",
     "l10n:extract": "yarn lingui:extract && yarn django:makemessages",
     "l10n:compile": "yarn lingui:compile && yarn django:compilemessages",
+    "l10n:garble": "node localebuilder.js --garble && yarn l10n:compile",
+    "l10n:ungarble": "git checkout -- 'locales/**/*.po' && yarn l10n:compile",
```

Comment:
Note that the newer way of doing the first part of this is `git restore`, but it's unavailable on the version of git that's in our container image, which is actually the one that's usually running this!  It definitely feels a bit weird to run a git command inside a dev container using a version of git that's likely different from the one on your host system, but uhh, it seems to work fine.